packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -21,18 +21,29 @@ const KEY_CODE_ESC = 27;
  * @extends ClayComponent
  */
 class ClayDropdownBase extends ClayComponent {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		this._eventHandler = new EventHandler();
+	}
+
 	/**
 	 * @inheritDoc
 	 */
 	attached() {
-		this.refs.portal.on('rendered', this._handleRenderedPortal.bind(this));
+		if (this.usePortal) {
+			this.refs.portal.on('rendered', this._alignAndFocusDropdown.bind(this));
+		}
 	}
 
 	/**
 	 * @inheritDoc
 	 */
-	created() {
-		this._eventHandler = new EventHandler();
+	rendered() {
+		if (!this.usePortal) {
```

Comment:
Just a suggestion: `rendered` is a hot path to updates, when `renderers` are always going to fall here (it will not align because of `this.expand`), but maybe we can improve this by calling in `syncExpanded`. What do you think?